src/collections/src/NICollectionViewModel.h

```diff
@@ -42,7 +42,8 @@
  *
  * @ingroup CollectionViewModels
  */
-@interface NICollectionViewModel : NSObject <NIActionsDataSource, UICollectionViewDataSource>
+@interface NICollectionViewModel : NSObject <NIActionsDataSource, UICollectionViewDataSource,
+    NSFastEnumeration>
```

Comment:
Where and how? I looked for a fair bit and I was unable to find a precedent. I've never seen this documented before, including in Apple's classes, and the behavior is strictly defined by the NSFastEnumeration protocol.

BTW, With http://nimbuskit.info down, it's enormously complicating for new users and new contributors.